.circleci/config.yml

```diff
@@ -1,21 +1,22 @@
 orbs:
-  slack: circleci/slack@3.4.1
+  slack: circleci/slack@3.4.2
+  node: circleci/node@4.1.0
 version: 2.1
 jobs:
   test:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-2004:202010-01
     steps:
       - checkout
-      - run: docker run --name postgres -p 5432:5432 -d postgres:10.6
+      - run: docker run --name postgres -e POSTGRES_PASSWORD='5ecr3t' -p 5432:5432 -d postgres:13
       - run:
           name: Install Node
           command: |
-            curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+            curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -
             sudo apt-get install -y nodejs
             echo 'export PATH=/usr/bin:$PATH' >> $BASH_ENV
             which node && node -v
-      - run: npm install
+      - node/install-packages
```

Comment:
is this working in a different way than `npm install`?

Reply:
It's doing npm ci with caching, but mostly I did it because it sounded fancy to use an orb. https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages

Reply:
Btw, that's also the reason I'm using the cimg Docker containers of CircleCI instead of the Ubuntu machine. Just because they sound fancy